modules/jt-extending-tekton-using-custom-tasks-and-scripts.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-extending-tekton-using-custom-tasks-and-scripts_{context}"]
```

Comment:
id="jt-extending-tekton-using-custom-tasks-and-scripts_{context}" --> id="jt-extending-tekton-capabilities-using-custom-tasks-and-scripts_{context}"